app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/manifestStatus.component.ts

```diff
@@ -0,0 +1,65 @@
+import { IComponentOptions, IController, IScope, module } from 'angular';
+import { KubernetesManifestService } from '../../../manifest/manifest.service';
+
+class KubernetesDeployManifestManifestStatusCtrl implements IController {
+  public manifestContents: any;
+  public stage: any;
+  public application: any;
+  public statuses: string;
+
+  private _manifest: any;
+
+  public get manifest() {
+    return this._manifest;
+  }
+
+  public set manifest(manifest) {
+    this._manifest = manifest;
+    this.statuses = '';
+    if (manifest.status == null) {
+      return;
+    }
+    const statuses = Object.keys(manifest.status).reduce((arr, key) => {
+      if (manifest.status[key].state) {
+        arr.push(key);
+      }
+      return arr;
+    }, []);
+    if (statuses.length > 0) {
+      this.statuses = '(' + statuses.join(', ') + ')';
+    }
+  }
+
+  constructor(private kubernetesManifestService: KubernetesManifestService, private $scope: IScope) {
+    'ngInject';
+
+    this.kubernetesManifestService.makeManifestRefresher(
```

Comment:
This will register a duplicate refresher, we already have one in the `kubernetes-show-manifest-details` directive. Any way we can merge those two?

Reply:
done